### PR TITLE
Adding new hud glasses and fixing rsi shit

### PR DESCRIPTION
**Pull request by Creatorbot01**
fucked up a bit and just close the previous pull cause im an idiot who forgot how to push shit
anyway

:cl:
- add: Added two new HUD glasses, medical and medsec

ADMIN:
- add: Added "centcom glasses", HUD sunglasses that provide welding protection as well as all three HUD types

### CODE REVIEW

**Comment by Creatorbot01** [comment]
nice removing the warden winter coat that rsi scream about work. it should be fine since wizden also didnt have the folder for it anyway.

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 1]
```diff
@@ -265,3 +265,68 @@
   - type: Clothing
     sprite: Clothing/Eyes/Glasses/ninjavisor.rsi
   - type: FlashImmunity
+
+- type: entity
+  parent: [ClothingEyesBase, ShowMedicalIcons]
+  id: ClothingEyesGlassesMedical
+  name: medical glasses
+  description: sunglasses that provide flash immunity and a vital monitoring of organic person.
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:273]:
```suggestion
  description: Upgraded sunglasses that provide flash immunity and a medical HUD.
```

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 2]
```diff
@@ -265,3 +265,68 @@
...
+    tags:
+    - HamsterWearable
+  - type: IdentityBlocker
+    coverage: EYES
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, ShowMedicalIcons, BaseSecurityCommandContraband]
+  id: ClothingEyesGlassesMedSec
+  name: medsec glasses
+  description: Upgraded sunglasses that provide flash immunity and also both medical scan and background check of the people around the user.
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:292]:
```suggestion
  description: Upgraded sunglasses that provide flash immunity, a medical HUD, and a security HUD.
```

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 3]
```diff
@@ -265,3 +265,68 @@
...
+    tags:
+    - HamsterWearable
+  - type: IdentityBlocker
+    coverage: EYES
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, BaseSecurityCommandContraband]
+  id: ClothingEyesGlassesCR
+  name: specialize NT glasses
+  description: A special sunglasses that protect user rentina from sudden flashing or sparking light and provide information regarding both structural and organic status. It also connected to "NT Security Server" for background check of nearby people.
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:311]:
```suggestion
  description: Upgraded sunglasses that provide flash immunity, welding protection, and all three main HUD types.
```

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 4]
```diff
@@ -265,3 +265,68 @@
...
+  - type: Tag
+    tags:
+    - HamsterWearable
+  - type: IdentityBlocker
+    coverage: EYES
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, BaseSecurityCommandContraband]
+  id: ClothingEyesGlassesCR
+  name: specialize NT glasses
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:310]:
```suggestion
  name: commander glasses
```

**Review by TheDoctor1977 — CHANGES_REQUESTED** [review]
standardize please

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml (hunk 4 above)
Comment (Creatorbot01) [reply Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:310]: To be fair this one is for all centcom agent but eh il rephrase

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml [hunk 5]
```diff
@@ -265,3 +265,68 @@
...
+    tags:
+    - HamsterWearable
+  - type: IdentityBlocker
+    coverage: EYES
+
+- type: entity
+  parent: [ClothingEyesBase, ShowSecurityIcons, BaseSecurityCommandContraband]
+  id: ClothingEyesGlassesCR
+  name: centcom glasses
+  description: A specially made sunglasses for Nanotransen field agent and official representative that provide both flash immunity and eye protection. It also come with with medical HUD, security HUD and diagnostic HUD.
```
Comment (TheDoctor1977) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:311]:
```suggestion
  description: Specially made sunglasses for Nanotransen field agents that provide both flash immunity and eye protection. They also provide a medical HUD, security HUD, and diagnostic HUD.
```

**Review by TheDoctor1977 — COMMENTED** [review]
going to see if i can just force fix this tiny grammar issue from my end

**Comment by TheDoctor1977** [comment]
i can go ahead and merge this now or it can wait until you add crafting recipes for them

**Comment by Creatorbot01** [comment]
nah im just gonna merge it. i can make another pr to add crafting recipe later

**Pull request merged**